common/header_language_selector/reports/pre-a11y-1.json

```diff
@@ -0,0 +1,444 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/sites/language/language-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Evaluation of how the Language Toggle Link meets accessibility requirements",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ""
```

Comment:
A CSS selector need to be defined.

Reply:
I am unsure what css selector could be defined here as the language selector works as a link <a> with an href of "/fr.html"